src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/_FileUpload.cshtml.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace SFA.DAS.AdminService.Web.Views.Roatp.Apply.Financial
+{
+    public class _FileUploadModel : PageModel
+    {
+        public void OnGet()
+        {
+        }
+    }
+}
```

Comment:
Do you really need this code behind? Also I'm not sure why it's be tacked onto a Razor View

Reply:
I have no idea where that came from, will delete.